<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestWindow</class>
 <widget class="QDialog" name="TestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>247</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Test Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Upload query image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblQueryImg">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="text">
       <string>Upload Vocabulary</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblVocabulary">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>541</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1">
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="text">
       <string>Upload histogram database</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lblHistDb">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate Test Statistics for All Test Data</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
